.golangci.yml

```diff
@@ -138,9 +138,10 @@ issues:
         - gosec
         - scopelint
         - unparam
+        - unused
```

Comment:
@cristiGuranIonos I assume that we are adding an exception for `unused` since there are a lot of false-positives in the tests and we don't care about them, right?